editor/project_settings_editor.cpp

```
@@ -722,7 +722,7 @@ ProjectSettingsEditor::ProjectSettingsEditor(EditorData *p_data) {
 
 	property_box = memnew(LineEdit);
 	property_box->set_placeholder(TTRC("Select a Setting or Type its Name"));
-	property_box->set_accessibility_name(TTRC("Setting Name"));
+	property_box->set_accessibility_name(TTRC("Select a Setting or Type its Name"));
```

Would it make sense to fall back to the placeholder text?

---

It already is using placeholder as fallback.

---

No, just missed it, it’s already removed.